lmms_eval/protocol.py

```
@@ -92,6 +96,8 @@ def to_openai_messages(self, video_kwargs: Dict[str, str] = None):
                 elif content.type == "image":
                     openai_message["content"].append({"type": "image_url", "image_url": {"url": f"data:image/png;base64,{self.encode_image(content.url)}"}})
                 elif content.type == "video":
+                    if fetch_video is None:
+                        raise ImportError("qwen_vl_utils is required for video processing. Please install it with: pip install qwen-vl-utils")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)